src/theme/widget.ts

```diff
@@ -1,5 +1,12 @@
 import { colorsDark, colorsLight } from './colors'
 
+const fonts = {
+  fontFamily: {
+    font: "'Inter custom', sans-serif;",
```

Comment:
Should this be `"'Inter custom';"`, without `sans-serif`? Based on https://github.com/Uniswap/interface/blob/main/public/index.html#L40.